File: TransCoder_tokenized_test_set_functions/java/PRINT_ARRAY_STRINGS_SORTED_ORDER_WITHOUT_COPYING_ONE_STRING_ANOTHER.java
static void printInSortedOrder ( String arr [ ], int n ) {
  int index [ ] = new int [ n ];
  int i, j, min;
  for ( i = 0;
  i < n;
  i ++ ) {
    index [ i ] = i;
  }
  for ( i = 0;
  i < n - 1;
  i ++ ) {
    min = i;
    for ( j = i + 1;
    j < n;
    j ++ ) {
      if ( arr [ index [ min ] ] . compareTo ( arr [ index [ j ] ] ) > 0 ) {
        min = j;
      }
    }
    if ( min != i ) {
      int temp = index [ min ];
      index [ min ] = index [ i ];
      index [ i ] = temp;
    }
  }
  for ( i = 0;
  i < n;
  i ++ ) {
    System . out . print ( arr [ index [ i ] ] + " " );
  }
}

File: TransCoder_tokenized_test_set_functions/python/PRINT_ARRAY_STRINGS_SORTED_ORDER_WITHOUT_COPYING_ONE_STRING_ANOTHER.py
def printInSortedOrder ( arr, n ) :
    index = [ 0 ] * n
    for i in range ( n ) :
        index [ i ] = i
    for i in range ( n - 1 ) :
        min = i
        for j in range ( i + 1, n ) :
            if ( arr [ index [ min ] ] > arr [ index [ j ] ] ) :
                min = j
        if ( min != i ) :
            index [ min ], index [ i ] = index [ i ], index [ min ]
    for i in range ( n ) :
        print ( arr [ index [ i ] ], end = " " )
    
